client/image_list.go

```diff
@@ -10,7 +10,14 @@ import (
 	"github.com/docker/docker/api/types/versions"
 )
 
+// ImageListWithManifests returns a list of images including the manifests information.
+
 // ImageList returns a list of images in the docker host.
+//
+// Experimental: Setting the [options.Manifest] will populate
```

Comment:
Bad rebase? (empty line and `ImageListWithManifests` -> `ImageList`)

Reply:
Ah, good catch!